### ISSUE TITLE
Assets do not work on .NET6 + WinUI 3 on mobile (Android)

### ISSUE THREAD

**Issue by MGRussell** [unoplatform/uno#8252]
### Current behavior

Trying to access assets through the filesystem, such as by: `Windows.Storage.StorageFile.GetFileFromApplicationUriAsync(new Uri(assetFileName.Text));` will fail with a file not found exception. Assets loaded through xaml such as by `<Image Source="/Assets/image.png"/>` are not affected by this, and do work.

### Expected behavior

Files should be accessible through code.

### How to reproduce it (as minimally and precisely as possible)

Create a new unoapp-winui-net6 project.
Add an asset.
Try to access it through code using something like: `Windows.Storage.StorageFile.GetFileFromApplicationUriAsync(new Uri(assetFileName.Text)` on Android.

I have created a repro project here: https://github.com/MGRussell/UnoAssetTestNet6WinUi which is just a WinUI3/.NET6 port of the sample project here: https://github.com/unoplatform/Uno.Samples/tree/master/UI/PackageResources

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### NuGet package version(s)

4.0.13 in the sample project. The issue was first discovered in a different project on version 4.2.0-dev.217

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

17.2.0 Preview 1

**Comment by jeromelaban** [comment]
Thanks for the report. Are you seeing some error messages in the logs ?

**Comment by MGRussell** [comment]
There's nothing that immediately stands out to me in the logs:

debug log - https://gist.github.com/MGRussell/879f441f6d7d915c1666839b94555803
build log - https://gist.github.com/MGRussell/36b814732bdfb8de3dcb97ca7430efa4
deployment log - https://gist.github.com/MGRussell/1333542d5ae23056c0687c8663f396ae

The numerous "[monodroid-assembly] open_from_bundles: failed to load assembly" log hits in the debug log occur in every Uno project for me and do not seem related to this.

**Comment by jeromelaban** [comment]
Thanks for the repro. The issue is related to the change in defaults for assets path for net6.0-android targets.

It'll be fixed by #8267, but you can work around it using:

```xaml
    <Target Name="test" BeforeTargets="_ComputeAndroidAssetsPaths">
        <CreateProperty Value="$(MonoAndroidAssetsPrefix);Assets">
            <Output
                TaskParameter="Value"
                PropertyName="MonoAndroidAssetsPrefix" />
        </CreateProperty>	
    </Target>
```

**Comment by MGRussell** [comment]
Excellent, thanks for the quick response/workaround. I have tested and it's working on my device. I'll leave this issue open in case you would prefer to close it after the fix is merged.
